atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/pool/CassandraClientPoolMetrics.java

```diff
@@ -93,6 +100,23 @@ public void recordPoolExhaustion() {
         poolExhaustionCounter.inc();
     }
 
+    public synchronized void recordEmptyPool() {
```

Comment:
Have to use `synchronized` here, as `getCount` and `inc` are atomic separately, not together :(